include/SFML/System/Utf.inl

```diff
@@ -56,7 +56,7 @@ In Utf<8>::decode(In begin, In end, std::uint32_t& output, std::uint32_t replace
 {
     // clang-format off
     // Some useful precomputed data
-    static constexpr int trailing[256] =
+    static constexpr std::array<std::uint8_t, 256> trailing =
```

Comment:
The template type was changed from `int` to `std::uint8_t` for two reasons. First of all using a smaller type saves memory. We don't need numbers bigger than `5` so `std::uint8_t` is a more efficient representation. Second, by using an unsigned type it avoids a sign conversion warning when using one of these array members to index into an array. C-style arrays let you index with signed types but `std::array` expects a `std::size_t`.